src/main/java/ListOfTasks.java

```diff
@@ -0,0 +1,130 @@
+import java.util.ArrayList;
+
+public class ListOfTasks {
+    private static int count;
+    private ArrayList<Task> xs;
+    private Ui ui;
+
+    public ListOfTasks() {
+        //this.xs = new Task[100];
+        ui = new Ui();
+        this.xs = new ArrayList<>();
+        this.count = 0;
+    }
+
+    public ArrayList<Task> getList() {
+        return xs;
+    }
+
+    public void includeAdditionalTask(Task x) {
+        xs.add(this.count,x);
+        count++;
+    }
+
+    public void addTask(String information) {
+        ui.addTaskMessage();
+        information = removeVal(information, "todo");
+        xs.add(count, new ToDo(information, "TODO"));
+        CompilationOfFiles.updateSavedFile(this.xs.get(count), "TODO");
+        ui.printCurrentTask(xs.get(count));
+        count++;
+        ui.printNumberOfTasks(count);
+    }
+
+    public void listOut() {
+        ui.listTaskMessage();
+        int a = 0;
+        while (a < count) {
+            ui.listEachTask(xs,a);
+            a = a + 1;
+        }
+    }
+
+    public void isDone(String command) {
```

Comment:
This is a method name, but don't you think the name isDone sounds more like a boolean variable name?